data/json/items/gun/308.json

```diff
@@ -19,7 +19,9 @@
     "recoil": 120,
     "durability": 7,
     "reload": 400,
-    "burst": 25,
+    "modes" : [
+       [ "DEFAULT", "auto", 25 ]
```

Comment:
This effectively disables semi-auto, which is I think what you intended and unless anyone disagrees I'd presume is correct?